includes/class-simple-local-avatars.php

```diff
@@ -316,12 +330,17 @@ public function avatar_settings_field( $args ) {
 			$this->options[ $args['key'] ] = 0;
 		}
 
-		echo '
+		if ( 'clear_cache' !== $args['key'] ) {
+			echo '
 			<label for="simple-local-avatars-' . esc_attr( $args['key'] ) . '">
 				<input type="checkbox" name="simple_local_avatars[' . esc_attr( $args['key'] ) . ']" id="simple-local-avatars-' . esc_attr( $args['key'] ) . '" value="1" ' . checked( $this->options[ $args['key'] ], 1, false ) . ' />
 				' . esc_html( $args['desc'] ) . '
 			</label>
 		';
+		} else {
+			echo '<button id="clear_cache_btn" class="button delete" name="clear_cache_btn" >' . esc_html__( 'Clear cache', 'simple-local-avatars' ) . '</button><br/>';
+			echo '<span id="clear_cache_message" style="font-style:italic;font-size:15px;line-height:2;"></span>';
```

Comment:
I think we should follow the font size of other field in the setting page.
```suggestion
			echo '<span id="clear_cache_message" style="font-style:italic;font-size:14px;line-height:2;"></span>';
```